refapp/SKWebApi/Skills/SystemPromptDefaults.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SKWebApi.Skills;
+
+internal static class SystemPromptDefaults
+{
+    internal const double tokenEstimateFactor = 2.5;
```

Comment:
In C#, constants start with a capital letter (please update all the names in this file)